bin/runtime/src/lib.rs

```diff
@@ -312,7 +313,7 @@ impl pallet_session::Config for Runtime {
     type ShouldEndSession = pallet_session::PeriodicSessions<SessionPeriod, Offset>;
     type NextSessionRotation = pallet_session::PeriodicSessions<SessionPeriod, Offset>;
     // The () SessionManager makes always a new session with the same set of validators as before.
-    type SessionManager = ();
+    type SessionManager = pallet_aleph::AlephSessionManager<Self>;
```

Comment:
I might be wrong here, but since the validator_list starts as None, doesn't it mean that at the genesis (block) there will be no validators? Maybe you should add some initial validators too, similar to how authorities are read from a file at genesis?

Reply:
Probably it should be added in chain_spec but I'm not sure. It depends if SessionManager is called before first session or not. I don't know.

Reply:
It should be, but I `None`, since currently it is `None`. @mike1729 if you will give me a list of validators I should put there I can add it into chain_spec. Just I don't know what values should there be.

Reply:
maybe you could just set it to the same accounts as authorities to start with?

Reply:
This should work but it would be good to know why, eg when the Session Manager is called for the first time. First session is special and there are several places where it's handled in a special way.